src/pressedAnotherOperator.js

```diff
@@ -0,0 +1,19 @@
+const pressedAnotherOperator = (button, hasLastChar) => {
```

Comment:
So where `pressedAnotherOperator` is called it is `button.key` that is passed into it.
But inside `pressedAnotherOperator`  ` we get the syntax error `Binding member expression` when trying to replace button with button.key. Unless you mean just changing `button` to `key` in which case yes that would work. Is that what you mean?

Reply:
@grimxreaper If you thought I meant to replace `button` with `button.key`, the error you describe makes sense. Passing in `button.key` means you don't have access to the button object anymore. I meant instead of naming the argument `button` you name it `key` since you are not passing in the button anymore, you're just passing in the key.